task2/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "task2",
+  "version": "1.0.0",
+  "description": "Task 2 react mentoring",
+  "main": "index.js",
+  "scripts": {
+    "start": "npm run dev",
+    "build": "npm run prod",
+    "prod": "./node_modules/.bin/webpack --config ./build/webpack.prod.config.js",
+    "dev": "./node_modules/.bin/webpack-dev-server --config ./build/webpack.dev.config.js",
```

Comment:
You can use npm installed module CLI without providing the relative or absolute path to the package.
E.g: 
"prod": "webpack --config ./build/webpack.prod.config.js",
"dev": "webpack-dev-server --config ./build/webpack.dev.config.js",

Reply:
Sure, I changed! thanks.